extensions/git/src/model.ts

```diff
@@ -118,6 +118,22 @@ export class Resource implements SourceControlResourceState {
 		}
 	}
 
+	private get tooltip(): string | undefined {
```

Comment:
Although smart, this automatic labelling based on the name of the enum value won't work with localize. Please create a private `getTooltip` method, similar to `getIconPath`, where the mapping from Status to tooltip is done manually.